### DISCUSSION TITLE
アンダースコア文字で終わるURLがリンクカードにならない

### DISCUSSION THREAD

**Discussion by shiracamus** [increments/qiita-discussions#518] [要望・アイデア・不具合]
下図のように、アンダーラインで終わるURLがリンクカード表示になりません。

![image](https://github.com/increments/qiita-discussions/assets/4151943/80bedd9a-e524-4b8c-9d40-9587fb577fbc)

**Comment by ohakutsu** [comment]
@shiracamus 
ご報告ありがとうございます！

QiitaのMarkdownはGitHub Flavored Markdownを拡張しているのですが、
GitHub Flavored Markdownの仕様で、リンクの末尾に`?`, `!`, `.`, `,`, `:`, `*`, `_`, `~`がある場合はその文字はリンクの一部として判定されず、リンクと通常のテキストが混ざった行として解釈される様です。
https://github.github.com/gfm/#autolinks-extension-

そのため、リンクカードとして表示できていないと思われます。

申し訳ないのですが、現時点ではお問い合わせいただいたURLをリンクカードとして表示することはできなさそうです :bow:
かわりに[リンクの記法](https://qiita.com/Qiita/items/c686397e4a0f4f11683d#links---%E3%83%AA%E3%83%B3%E3%82%AF) をお使いいただけると幸いです :pray:

**Reply by shiracamus** [reply]
なるほど。それでは仕方がないですね。
リンクの記法を使うことにします。
ありがとうございました。
